perceval/backends/core/googlehits.py

```diff
@@ -130,7 +130,7 @@ def has_resuming(cls):
     def metadata_id(item):
         """Extracts the identifier from a GoogleHit item."""
 
-        return item['id']
+        return item['id'] + '-' + str(item['fetched_on'])
```

Comment:
Is this the way we do this in other backends? Do we concat fields using a `- `?

Reply:
we generally have just the `id` without any concatenation. I can modify the googlehits `__uuid` method to include the `fetched_on` value. What do you think?

Reply:
I have modifed the PR, now the method `__uuid` takes care of joining the `fetched_on` attribute.